api/src/main/resources/liquibase.xml

```diff
@@ -793,4 +793,707 @@
+			FROM concept
+			WHERE uuid = 'cbf0fa45-cfb3-4777-ab14-7eb187df7c1b';
+
+			INSERT INTO concept_reference_term(concept_source_id, `code`, creator, date_created, retired, uuid)
+			SELECT c_source.concept_source_id, 'VLVAP', 1, CURRENT_TIMESTAMP, '0', '321832c6-2b5c-41dc-b2fe-6bd3206c34a7'
+			FROM concept_reference_source c_source
+			WHERE c_source.uuid = 'f124da75-9f50-4b25-be97-13c029e3a65b';
+
+			INSERT INTO concept_reference_map (concept_id, concept_reference_term_id, concept_map_type_id, date_created, creator, uuid)
+			SELECT c.concept_id, LAST_INSERT_ID(), '2', NOW(), 1, '076cbdaf-d9e6-428b-8f0c-3efb3d8910b9'
+			FROM concept c
+			WHERE c.uuid = 'cbf0fa45-cfb3-4777-ab14-7eb187df7c1b';
+
+			INSERT INTO concept_reference_term(concept_source_id, `code`, creator, date_created, retired, uuid)
+			SELECT c_source.concept_source_id, 'LCVAP', 1, CURRENT_TIMESTAMP, '0', '35b21e9e-a957-45eb-b8bc-0b45bbe7de55'
+			FROM concept_reference_source c_source
+			WHERE c_source.uuid = 'c19e926e-fc28-4d78-9b99-eb42032abf3c';
+
+			INSERT INTO concept_reference_map (concept_id, concept_reference_term_id, concept_map_type_id, date_created, creator, uuid)
+			SELECT c.concept_id, LAST_INSERT_ID(), '2', NOW(), 1, '6a1b667a-f422-4f50-8b40-c349384225f0'
+			FROM concept c
+			WHERE c.uuid = 'cbf0fa45-cfb3-4777-ab14-7eb187df7c1b';
+
+			-- VIH-1 Charge Virale RNA PCR (CV-ce) Plasma
+			INSERT INTO concept (uuid, retired, datatype_id, class_id, creator, date_created)
+			SELECT 'ba65bd79-a9e2-4372-9515-ee88039850d9', '0', c_datatype.concept_datatype_id, c_class.concept_class_id, '1', CURRENT_TIMESTAMP
+			FROM concept_datatype c_datatype INNER JOIN  concept_class c_class
+			WHERE c_datatype.hl7_abbreviation = 'CWE' AND c_class.uuid = '8d4907b2-c2cc-11de-8d13-0010c6dffd0f';
+
+			INSERT INTO concept_name (concept_id, name, locale, creator, date_created, voided, locale_preferred, uuid)
+			SELECT concept_id, 'VIH-1 Charge Virale RNA PCR (CV-ce) Plasma', 'en-GB', '1', CURRENT_TIMESTAMP, '0', '1', '47db4b76-1d12-4025-b6a0-3745215700a2'
+			FROM concept
+			WHERE uuid = 'ba65bd79-a9e2-4372-9515-ee88039850d9';
+
+			INSERT INTO concept_reference_term(concept_source_id, `code`, creator, date_created, retired, uuid)
+			SELECT c_source.concept_source_id, 'VLVCP', 1, CURRENT_TIMESTAMP, '0', '56b88223-bbc8-4741-8a73-124b19019770'
+			FROM concept_reference_source c_source
+			WHERE c_source.uuid = 'f124da75-9f50-4b25-be97-13c029e3a65b';
+
+			INSERT INTO concept_reference_map (concept_id, concept_reference_term_id, concept_map_type_id, date_created, creator, uuid)
+			SELECT c.concept_id, LAST_INSERT_ID(), '2', NOW(), 1, '06e90ae4-f1db-44c4-8642-6d9bde15801a'
+			FROM concept c
+			WHERE c.uuid = 'ba65bd79-a9e2-4372-9515-ee88039850d9';
+
+			INSERT INTO concept_reference_term(concept_source_id, `code`, creator, date_created, retired, uuid)
+			SELECT c_source.concept_source_id, 'LCVCP', 1, CURRENT_TIMESTAMP, '0', 'fabbf694-58cb-4009-9157-158b5080cb7a'
```

Comment:
I've checked the documentation and you should add here 'LCVAP' instead of 'LCVCP'